applications/serial_lte_modem/doc/TWI_AT_commands.rst

```diff
@@ -0,0 +1,246 @@
+.. _SLM_AT_TWI:
+
+TWI AT commands
+***************
+
+.. contents::
+   :local:
+   :depth: 2
+
+The following commands list contains AT commands related to the two-wire interface (TWI).
+
+Open TWI instance #XTWILS
+=========================
+
+The ``#XTWILS`` command lists all available TWI instances.
+
+Set command
+-----------
+
+The set command allows you to list all available TWI instances.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XTWILS: <index>[[[,<index>],<index>],<index>]
+
+The ``<index>`` parameter maps to the below TWI instance:
+
+* ``0`` - Use TWI0 (``i2c0``).
+* ``1`` - Use TWI1 (``i2c1``).
+* ``2`` - Use TWI2 (``i2c2``).
+* ``3`` - Use TWI3 (``i2c3``).
+
+See :ref:`zephyr:dtbinding_nordic_nrf_twi`.
+
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+There is no response.
+
+Example
+~~~~~~~
+
+::
+
+   AT#XTWILS
+   #XTWILS: 2
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+Write to TWI slave device #XTWIW
```

Comment:
Can we replace slave device, for example, with "peripheral device"?

Reply:
Yes peripheral is fine. Modified per suggestion.